.github/workflows/release.yml

```
@@ -0,0 +1,91 @@
+
+      - name: Generate changelog
+        id: changelog
+        run: |
+          # Get commits since last tag
+          if git describe --tags --abbrev=0 >/dev/null 2>&1; then
+            LAST_TAG=$(git describe --tags --abbrev=0)
+            CHANGELOG=$(git log --pretty=format:"- %s (%an)" $LAST_TAG..HEAD --no-merges)
+          else
+            CHANGELOG=$(git log --pretty=format:"- %s (%an)" --no-merges)
+          fi
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)